tests/conftest.py

```diff
@@ -23,13 +23,21 @@
 # see composer.utils.reproducibility.configure_deterministic_mode
 os.environ["CUBLAS_WORKSPACE_CONFIG"] = ":4096:8"
 
+# during the pytest refactor transition, this flag
+# indicates whether to include the deprecated fixtures
+include_deprecated_fixtures = True
```

Comment:
I don't think having this as a flag is needed (let's include deprecated fixtures right now), but if we want to keep it as a flag, can it be moved to a CLI argument?

Reply:
It's more for internal dev purposes, so that we can switch the mode to `False`, and know what is broken once we deprecate the old fixtures. I'll make this a private variable, would that work?